public/js/app/validation.js

```diff
@@ -0,0 +1,43 @@
+const api_data = "https://api.faforever.com/data/player"
```

Comment:
There is an env variable API_URL pointing to https://api.faforever.com (or the test server, or localhost depending on your environment). This must be used to be environment-agnostic.

Reply:
also this would be a Cross Site Request which is proabably not gonna work